src/routes/(app)/home/+page.server.ts

```diff
@@ -57,7 +57,7 @@ export const actions = {
 				body: "Couldn't create project"
 			};
 		} else {
-			track('Prompt Created', { user_id: session.user.id });
+			trackEvent('Prompt Created', undefined, { user_id: session.user.id });
```

Comment:
e.g. here you can then just do `trackEvent('Prompt Created', { user_id: session.user.id })`